questionnaire.markdown

```diff
@@ -34,5 +34,7 @@ For your convenience, a copy of the questionnaire's questions is quoted here in
 >      Considerations" sections?
 > 16.  Do features in your specification enable origins to downgrade default
 >      security protections?
-> 17.  How does your feature handle non-"fully active" documents?
```

Comment:
I think we need to keep this question and add a question around BFCache.

Whether we should add state to documents specifically for BFCache depends on whether APIs will need that information. I suspect most will just need to care for "fully active" and the potential for that to change over time.

Reply:
I added this question to the doc and was specifically targeting BFCache, so I think we should just update this question. Maybe we also want to call out the two cases of BFCache and detached iframes explicitly (but mostly focus on BFCache because it has its own guide)?

(replied on the other thread for "fully active" vs "bfcache" to keep that discussion in that thread)

Reply:
BFCache is a good reason for adding it, but there are other reasons besides BFCache to ask it (e.g., removal of an `iframe` element) so I don't think we should conflate them going forward.

Instead we should separately ask about BFCache as that has other implications.

Reply:
Hmm, ok, but I don't know if the detached iframe case question fits in a security & privacy questionnaire though since there's no way to reuse the document. If we were to add it, what do you think it should mention? My rough idea right now is:

What happens when a document that uses your feature gets detached?
When an iframe containing a document gets detached, the document is no longer fully active. The document is gone from the user's perspective, and should probably be treated as such by your feature too.
[not really sure what other advice to give, maybe point to the bfcache guide too for the advices that fit this case...]

Reply:
Alright, added in the latest patch!